build_variables.bzl

```diff
@@ -592,6 +592,10 @@ libtorch_nativert_sources = [
     "torch/nativert/graph/TensorMeta.cpp",
 ]
 
+libtorch_nativert_standalone_headers = [
+    "torch/nativert/executor/ExecutorConfig.h",
+]
+
```

Comment:
Not an expert on our build infra but I feel we don't need to special handle headers in the cmake?

Usually we have a catch-all rule to automatically the headers.

Reply:
Oh probably for buck I ran into a similar issue and I fixed in with `//caffe2:libtorch` which should include all the headers now, see [D74184245](https://www.internalfb.com/diff/D74184245)